app/Notifications/TwitterNotSent.php

```diff
@@ -25,7 +24,7 @@ public function __construct(string $error, Status $status) {
     }
 
     /** @deprecated will be handled in frontend */
-    public static function render(DatabaseNotification $notification): ?string {
+    public static function render(mixed $notification): ?string {
```

Comment:
mhmmhhm. Can you be more specific please?

Reply:
Nope, does not work because of the calling in the `Backend\NotificationController`.